en/docs/remote-apps/connect-remote-applications.md

```diff
@@ -0,0 +1,118 @@
+# Connect Remote Applications
+
+If you need to observe an external application that you have written in [Ballerina programming language](https://ballerina.io/), you can connect it to Choreo.
+
+## Prerequisites
+
+Create an application in Ballerina programming language. For instructions, see [Ballerina Documentation - Writing Your First Ballerina Program](https://ballerina.io/learn/user-guide/getting-started/writing-your-first-ballerina-program/).
+
+!!! Tip
+    In this guide, we are using the same Ballerina application that is created in the guide linked above.
+
+## Step 1: Generate and copy the Ballerina configurations
+
+In this step, you will generate the Ballerina configurations that include an application secret to link the remote application to Choreo.
+
+To get these configurations, follow this procedure:
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/).
+
+2. Go to the **Remote Apps** card, and click **Connect App**.
+
+3. Click **Connect**.
+
+4. In the **Connect existing application** card, enter a name for the remote application (e.g., `HelloService`).
+
+5. Click **Next**. The Ballerina configurations with the application secret generated for the remote application are displayed.
+
+6. Copy the Ballerina configurations.
+
+Now you have the application secret that you need to set up the Ballerina project of your remote application.
+
+## Step 2: Set up the Ballerina project
+
+In this step, you will set up your Ballerina project with the configurations that connect the Ballerina application to Choreo. Here, the Ballerina project is the directory that contains the definition of your Ballerina application in a `.bal` file and all the configuration files required to run it.
+
+To set up the Ballerina Project:
```

Comment:
We have not capitalized the P with regard to the term Project in the heading and para above. So I guess we can avoid the capitalization here too.

Reply:
Corrected